tests/kafkatest/services/kafka/kafka.py

```diff
@@ -1362,7 +1356,7 @@ def alter_message_format(self, topic, msg_format_version, node=None):
     def kafka_acls_cmd_with_optional_security_settings(self, node, force_use_zk_connection, kafka_security_protocol = None, override_command_config = None):
         if self.quorum_info.using_kraft and not self.quorum_info.has_brokers:
             raise Exception("Must invoke kafka-acls against a broker, not a KRaft controller")
-        force_use_zk_connection = force_use_zk_connection or not self.all_nodes_acl_command_supports_bootstrap_server
```

Comment:
The only place where we were using (now removed) `all_nodes_acl_command_supports_bootstrap_server`